src/atoms/Spacer/index.js

```diff
@@ -0,0 +1,67 @@
+import React, { PropTypes } from 'react';
+import classnames from 'classnames';
+
+import styles from './spacer.module.scss';
+
+function Spacer ( props ) {
+  const {
+    xSmall,
```

Comment:
I think this is a great example of when explicit prop types should be used instead of `type='...'` - really, a finite number of options instead of a potentially ever-growing set of component variations